okapi/core/Cache.php

```diff
@@ -129,10 +123,13 @@ public static function get_many($keys)
                     "Length of data, compressed: ".strlen($row['value']));
             }
         }
-        if (count($dict) < count($keys))
-            foreach ($keys as $key)
-                if (!isset($dict[$key]))
+        if (count($dict) < count($keys)) {
+            foreach ($keys as $key) {
+                if (!isset($dict[$key])) {
                     $dict[$key] = null;
+                }
+            }
+        }
```

Comment:
Lots of boilerplate code introduced here - seven code lines instead of three, without any improvement to readability.

Reply:
My comment is not undestandible without the previous lines. Suggest to open a separate issue for discussion.